extensions/widgets/segmented/notes/18319.md

```diff
@@ -0,0 +1 @@
+# Segmented control fill was bleeding outside of the border.
```

Comment:
@trevordevore Argh! I need to document the two release note formats... the LCB release notes and extension release notes use a different format of release note, where a bugfix needs to be notated using:

```markdown
# [18319] Segmented control fill was bleeding outside of the border.
```

See [this example](https://github.com/livecode/livecode/blob/develop/extensions/widgets/navbar/notes/16399.md)